backend/main.py

```
@@ -1,369 +1,305 @@
-from fastapi import WebSocket, WebSocketDisconnect, Depends
-from typing import Dict, List
-
-# A simple in-memory manager for active WebSocket connections.
-# For production, a more robust solution (e.g., Redis-based) would be needed for scalability.
-class ConnectionManager:
-    def __init__(self):
-        # rooms: { "room_id": [WebSocket, WebSocket, ...] }
-        self.active_connections: Dict[str, List[WebSocket]] = {}
+from fastapi import WebSocket, WebSocketDisconnect
 
+class ChatConnectionManager:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)